src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -19,11 +20,12 @@ public Movie add(Movie movie) {
 
     @Override
     public Movie get(Long id) {
-        return movieDao.get(id).get();
+        return movieDao.get(id).orElseThrow(() ->
+                new EntityNotFoundException("Can't get movie by id " + id));
```

Comment:
According to the checklist, the `get()` method on the DAO layer should return `Optional`, and the entity should be returned on the service layer. Here, you have correctly implemented the service layer to return the entity and handle the case where the movie is not found. Well done!